src/CampaignKit.PortfolioImporter/Entities/Character.cs

```diff
@@ -45,8 +47,22 @@ public class Character
         /// <value>The XML.</value>
         public string Xml { get; set; }
 
+		/// <summary>
+		///		Gets or sets the game system
+		/// </summary>
         public string Game { get; set; }
 
-        #endregion
-    }
+		#endregion
+
+		#region Abstract Members
+
+		/// <summary>
+		/// Return the character formatted in the system default format.
+		/// </summary>
+		/// <returns></returns>
+		public abstract string getDefaultFormat();
```

Comment:
Please use PascalCasing for Namespaces, Types, Properties, Methods, and camelCasing for parameters and fields.

Reply:
Nevermind, I can do the formatting afterwards.